### PR TITLE
Add log filename hook to krel/release-notes

### PR DESCRIPTION
**Pull request by saschagrunert**
The log filename hook adds a new field to every log indicating the file
location of the log:

```
INFO my message  file="notes/notes.go:520"
```

### CODE REVIEW

File: pkg/log/hooks.go [hunk 1]
```diff
@@ -0,0 +1,130 @@
+/*
+Copyright 2019 The Kubernetes Authors.
```
Comment (hoegaarden) [inline pkg/log/hooks.go]:
```suggestion
Copyright 2020 The Kubernetes Authors.
```

🤷‍♂

Comment (saschagrunert) [reply pkg/log/hooks.go]: Fixed :+1:

**Pull request merged**